iac-extensions/cloudformation/src/test/java/org/sonar/iac/cloudformation/reports/CfnLintImporterTest.java

```diff
@@ -56,19 +56,20 @@ class CfnLintImporterTest {
 
   @BeforeEach
   void setUp() throws IOException {
-    File baseDir = new File("src/test/resources/cfn-lint");
+    File baseDir = new File(PATH_PREFIX);
     context = SensorContextTester.create(baseDir);
 
-    File someFile = new File("src/test/resources/cfn-lint/template.yaml");
+    File someFile = new File(PATH_PREFIX + "/template.yaml");
     context.fileSystem().add(new TestInputFileBuilder("project", baseDir, someFile).setContents(new String(Files.readAllBytes(someFile.toPath()))).build());
```

Comment:
It's used in many places. Let's refactor it in scope of [SONARIAC-704](https://sonarsource.atlassian.net/browse/SONARIAC-704)